docs/fsharp/tour.md

```diff
@@ -21,7 +21,7 @@ There are two primary concepts in F#: functions and types.  This tour will empha
 ## How to Run the Code Samples
 
 >[!NOTE]
-Eventually, this documentation site will have the ability to run these code samples directly in your browser.  If your browser supports Silverlight, you can use [TryFsharp](http://www.tryfsharp.org/Create) for the time being.
+Eventually, this documentation site will have the ability to run these code samples directly in your browser. You can choose two way: Try [F# for Azure Notebooks](https://notebooks.azure.com/Microsoft/libraries/fsharp/html/FSharp%20for%20Azure%20Notebooks.ipynb) on Microsoft Azure Notbooks (Currently preview, free to use.) Or, if your browser supports Silverlight, you can use [TryFsharp](http://www.tryfsharp.org/Create) for the time being.
```

Comment:
> Eventually, this documentation site will have the ability to run these code samples directly in your browser.

Side note: We removed that statement from the csharp version. It's been lingering since early 2016. There is no clear ETA on when it would be available. Recommendation: Remove that sentence here, too.

It's not quite an either/or situation, is it? I mean someone could play with both of these approaches.

Here's a few things to just consider for the passage whatever it turns out to be:
* I suggest dropping the "Or"
* "You can" language isn't recommended by the style manual.
* I'd reverse the order.
* There's a spelling error for "Notbooks."
* It's tough to keep up with features of Azure currently in preview and get the doc adjusted immediately when it comes off of preview.
* It's not wise for us to talk about "free" things in the docs ... it incurs overhead (e.g., mentioning that bandwidth charges apply, referencing the license, keeping up with the product when it's no longer free, we can only use the word "free" for six months for any given product/service).

How about something like this for the note ...

> Two options for running the code samples are [TryFsharp](http://www.tryfsharp.org/Create) (requires Silverlight) and [F# for Azure Notebooks](https://notebooks.azure.com/Microsoft/libraries/fsharp/html/FSharp%20for%20Azure%20Notebooks.ipynb) on Microsoft Azure.

cc/ @cartermp